### PR TITLE
Revert 1.2 additions

### PR DESCRIPTION
**Pull request by tovganesh**
to prepare for an intermediate release 1.1.7, revert changes pertaining to 1.2 for now.

### CODE REVIEW

**Comment by TapasSenapati** [comment]
Is dbt run succeeding with a demo project now.

**Comment by tovganesh** [comment]
> Is dbt run succeeding with a demo project now.

Yes, attaching runs here with a demo dbt project:

ganesh.venkateshwara@MacBook-Pro dbtdemo % dbt debug
16:41:45  Running with dbt=1.1.2
dbt version: 1.1.2
python version: 3.9.11
python path: /Users/ganesh.venkateshwara/.pyenv/versions/3.9.11/bin/python
os info: macOS-12.6-x86_64-i386-64bit
Using profiles.yml file at /Users/ganesh.venkateshwara/.dbt/profiles.yml
Using dbt_project.yml file at /Users/ganesh.venkateshwara/code/dbtdemo/dbt_project.yml

Configuration:
  profiles.yml file [OK found and valid]
  dbt_project.yml file [OK found and valid]

Required dependencies:
 - git [OK found]

Connection:
  host: localhost
  port: 21050
  schema: dbtdemo
  username: None
  Connection test: [OK connection ok]

All checks passed!
ganesh.venkateshwara@MacBook-Pro dbtdemo % dbt docs generate
16:41:58  Running with dbt=1.1.2
16:41:58  Found 3 models, 4 tests, 0 snapshots, 0 analyses, 191 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics
16:41:58  
16:41:58  Concurrency: 4 threads (target='dev_impala_local')
16:41:58  
16:41:58  Done.
16:41:58  Building catalog
16:41:58  Catalog written to /Users/ganesh.venkateshwara/code/dbtdemo/target/catalog.json
ganesh.venkateshwara@MacBook-Pro dbtdemo % dbt run
16:58:41  Running with dbt=1.1.2
16:58:41  Found 3 models, 4 tests, 0 snapshots, 0 analyses, 191 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics
16:58:41  
16:58:42  Concurrency: 4 threads (target='dev_impala_local')
16:58:42  
16:58:42  1 of 3 START table model dbtdemo.my_first_dbt_model ............................ [RUN]
17:01:20  1 of 3 OK created table model dbtdemo.my_first_dbt_model ....................... [OK in 158.48s]
17:01:20  2 of 3 START incremental model dbtdemo.my_incremental_model .................... [RUN]
17:01:20  3 of 3 START view model dbtdemo.my_second_dbt_model ............................ [RUN]
17:02:22  3 of 3 OK created view model dbtdemo.my_second_dbt_model ....................... [OK in 61.40s]
17:06:42  2 of 3 OK created incremental model dbtdemo.my_incremental_model ............... [OK in 322.27s]
17:06:42  
17:06:42  Finished running 1 table model, 1 view model, 1 incremental model in 480.98s.
17:06:42  
17:06:42  Completed successfully
17:06:42  
17:06:42  Done. PASS=3 WARN=0 ERROR=0 SKIP=0 TOTAL=3
ganesh.venkateshwara@MacBook-Pro dbtdemo % dbt seed 
06:00:22  Running with dbt=1.1.2
06:00:22  Found 3 models, 4 tests, 0 snapshots, 0 analyses, 191 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics
06:00:22  
06:00:26  Concurrency: 4 threads (target='dev_impala_local')
06:00:26  
06:00:26  1 of 1 START seed file dbtdemo.sample_seed ..................................... [RUN]
06:01:26  1 of 1 OK loaded seed file dbtdemo.sample_seed ................................. [INSERT 3 in 60.63s]
06:01:26  
06:01:26  Finished running 1 seed in 64.14s.
06:01:26  
06:01:26  Completed successfully
06:01:26  
06:01:26  Done. PASS=1 WARN=0 ERROR=0 SKIP=0 TOTAL=1
ganesh.venkateshwara@MacBook-Pro dbtdemo % dbt test
06:01:58  Running with dbt=1.1.2
06:01:58  Found 3 models, 4 tests, 0 snapshots, 0 analyses, 191 macros, 0 operations, 1 seed file, 0 sources, 0 exposures, 0 metrics
06:01:58  
06:01:58  Concurrency: 4 threads (target='dev_impala_local')
06:01:58  
06:01:58  1 of 4 START test not_null_my_first_dbt_model_id ............................... [RUN]
06:01:58  2 of 4 START test not_null_my_second_dbt_model_id .............................. [RUN]
06:01:58  3 of 4 START test unique_my_first_dbt_model_id ................................. [RUN]
06:01:58  4 of 4 START test unique_my_second_dbt_model_id ................................ [RUN]
06:01:58  2 of 4 PASS not_null_my_second_dbt_model_id .................................... [PASS in 0.13s]
06:02:01  1 of 4 FAIL 1 not_null_my_first_dbt_model_id ................................... [FAIL 1 in 3.00s]
06:02:01  3 of 4 PASS unique_my_first_dbt_model_id ....................................... [PASS in 3.33s]
06:02:02  4 of 4 PASS unique_my_second_dbt_model_id ...................................... [PASS in 3.96s]
06:02:02  
06:02:02  Finished running 4 tests in 4.20s.
06:02:02  
06:02:02  Completed with 1 error and 0 warnings:
06:02:02  
06:02:02  Failure in test not_null_my_first_dbt_model_id (models/example/schema.yml)
06:02:02    Got 1 result, configured to fail if != 0
06:02:02  
06:02:02    compiled SQL at target/compiled/dbtdemo/models/example/schema.yml/not_null_my_first_dbt_model_id.sql
06:02:02  
06:02:02  Done. PASS=3 WARN=0 ERROR=1 SKIP=0 TOTAL=4

File: tests/conftest.py [hunk 1]
```diff
@@ -8,15 +8,16 @@
...
         'threads': 5,
+        'dbname': 'dbttest',
         'schema': 'dbttest',
         'host': 'coordinator-dbt-impala.dw-ciadev.cna2-sx9y.cloudera.site',
         'http_path': 'cliservice',
         'port': 443,
         'auth_type': 'ldap',
         'use_http_transport': True,
         'use_ssl': True,
-        'user': os.getenv('IMPALA_USER'),
```
Comment (TapasSenapati) [inline tests/conftest.py:18]: was this intended

Comment (tovganesh) [reply tests/conftest.py:18]: good catch. 
unintended consequence of revering.

Comment (tovganesh) [reply tests/conftest.py:18]: fixed this

**Review by TapasSenapati — APPROVED** [review]
Looks good.

**Pull request merged**